### ISSUE TITLE
Crash in 4.5.0 (9016) when Creating Entry

### ISSUE THREAD

**Issue by chanomie** [redbug26/kypass#2]
This is consistently happening for me for the last few months of TestFlight builds.
Currently running: 4.5.0 (9016)
iPad Pro running 12.2 Public Beta 3
iPhone XS running 12.2 Public Beta 3
Key file is stored in Dropbox

1. Open KyPass
2. Open key file and authenticate (either with password or Face ID)
3. Tap on +
4. Tap on Add Entry

App crashes

This affects two different keychain files I have bathed synched via dropbox.

**Comment by redbug26** [comment]
I couldn't reproduce myself with current release of iOS (I don't have public beta 3). I hope it's not an error from the beta iOS :(
Do you always have the problem ? Or just some times ?

**Comment by chanomie** [comment]
Attached is the crashlog from system:
```
Last Exception Backtrace:
(0x1e4df46ec 0x1e3ffbd00 0x1e4d0a148 0x1e57e37a0 0x2125c5c68 0x2125dc720 0x100739594 0x1005e0c9c 0x1005dfaa4 0x211e04960 0x211d71784 0x211d71a94 0x211d729d8 0x211d73cb0 0x211d56398 0x21287a0f4 0x1e3ff7454 0x1e94c3900 0x1e94c3bec 0x1e942ba98 0x1e945a734 0x1e945b4c0 0x1e4d83d5c 0x1e4d7e84c 0x1e4d7edb0 0x1e4d7e5d0 0x1e6fbadd4 0x2123d4424 0x100623400 0x1e483d2b4)

Thread 0 Crashed:
0   libsystem_kernel.dylib        	0x00000001e49890bc 0x1e4965000 + 147644
1   libsystem_c.dylib             	0x00000001e48e2184 0x1e4889000 + 364932
2   libc++abi.dylib               	0x00000001e3fe3ff0 0x1e3fe2000 + 8176
3   libc++abi.dylib               	0x00000001e3fe41b4 0x1e3fe2000 + 8628
4   libobjc.A.dylib               	0x00000001e3ffc144 0x1e3ff6000 + 24900
5   libc++abi.dylib               	0x00000001e3ff09d4 0x1e3fe2000 + 59860
6   libc++abi.dylib               	0x00000001e3ff05cc 0x1e3fe2000 + 58828
7   libobjc.A.dylib               	0x00000001e3ffbee0 0x1e3ff6000 + 24288
8   CoreFoundation                	0x00000001e4d7e63c 0x1e4cd7000 + 685628
9   GraphicsServices              	0x00000001e6fbadd4 0x1e6fb0000 + 44500
10  UIKitCore                     	0x00000002123d4424 0x211adf000 + 9393188
11  KyPass                        	0x0000000100623400 0x1005c0000 + 406528
12  libdyld.dylib                 	0x00000001e483d2b4 0x1e483c000 + 4788
```
[KyPass-2019-02-23-080725..ips.txt](https://github.com/redbug26/kypass/files/2898086/KyPass-2019-02-23-080725.ips.txt)

**Comment by redbug26** [comment]
Fixed in v4.5.1 (bug with old format of keepass database - kdb version 1)

**Comment by chanomie** [comment]
Confirmed it’s fixed!
